src/components/utils/hooks/useMessagePreviews.tsx

```diff
@@ -23,75 +26,79 @@ const useMessagePreviews = () => {
   const setPreviewMessages = useMessageStore((state) => state.setPreviewMessages);
   const [messagesLoading, setMessagesLoading] = useState<boolean>();
 
-  const peerAddresses = Array.from(conversations.keys());
-  const request = { ownedBy: peerAddresses, limit: 50 };
-  const {
-    loading: profilesLoading,
-    error: profilesError,
-    fetchMore
-  } = useQuery(ProfilesDocument, {
+  const profileIds = Array.from(conversations.keys())
```

Comment:
I don't love having to jump through all these hoops to get the `profileIds`. Might replace with a `Set<string>` to store all the `profileIds` discovered